app/src/androidTest/java/com/duckduckgo/app/tabs/store/TabStatsBucketingTest.kt

```diff
@@ -0,0 +1,313 @@
+package com.duckduckgo.app.tabs.store
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.duckduckgo.app.tabs.model.TabRepository
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.kotlin.whenever
+
+class DefaultTabStatsBucketingTest {
```

Comment:
This doesn’t have to be an Android Test, we can move it to unitTest instead and run it as such.

Reply:
Good point!